src/porepy/models/momentum_balance.py

```diff
@@ -795,11 +795,7 @@ def contact_mechanics_numerical_constant(
             c_num: Numerical constant.
 
         """
-
-        # Physical interpretation (IS):
-        # As a crude way of making the fracture softer than the matrix, we scale by
-        # one order of magnitude.
-        # Alternative interpretation (EK):
+        # Interpretation (EK):
```

Comment:
Do you think the remaining interpretation is useful, or should it go as well? I have no particular opinions, implement or ignore and resolve as you see fit.